MAINTAINERS

```diff
@@ -85,13 +85,7 @@ R:	Pankaj Gupta <pankaj.gupta@nxp.com>
 S:	Maintained
 F:	core/arch/arm/plat-ls/
 
-NXP (Freescale) i.MX6 Quad SABRE Lite, i.MX6 Quad SABRE SD
-R:	Yan Yan <yan.yan@windriver.com>
-R:	Feng Yu <Yu.Feng@windriver.com>
-S:	Maintained
-F:	core/arch/arm/plat-imx/
-
-NXP (Freescale) i.MX6 UltraLite EVK, i.MX6 ULL EVK, i.MX6 SX Sabreauto, i.MX7 Dual SabreSD
+NXP (Freescale) i.MX Family
```

Comment:
s/Family/family/ and in the commit message and subject: s/Latest/latest/ s/Mark/mark/ s/Maintainer/maintainer/ s/Family/family/ s/readme/README.md/ s/maintainer/MAINTAINERS/